### DISCUSSION TITLE
Swell Payment Gateway

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#418] [Help]
Hi, i was wondering if Swell has its own payment Checkout gateway, or is it just the test one they have.
I know swell has the test payment checkout gateway where you can test your checkout, but i wasnt sure how to add my bank account to swell so that the payment made comes to my account. So i wanted to know if the only way we achieve this is by using a 3rd party gateway integrated with swell like stripe, to get this feature?

**Comment by sags95** [comment]
Hi @Daniel-Alamezie, we currently only have the Test Gateway which can be used to test the checkout flow. You would have to select one of the supported gateways in Settings > Payments and select either Connect Stripe or choose another gateway.

**Comment by Daniel-Alamezie** [comment]
Hi,

Thank you for your reply and the clarification.

I guess my next question is, does swell provide a login front for
authenticating and authorising users, or do they just provide a way to
create an account so to track sessions of a specific cart?

Thanks

**Reply by sags95** [reply]
We do provide methods with swell-js to create accounts and manage sessions, you would just have to build a UI around these.

**Comment by Daniel-Alamezie** [comment]
Also If i connected stripe as the gateway and the user gets sent the a reminder message for the items in the basket, if they click on that link to confirm and finish payment, will the payment go through swell or through stripe. 
For context, this is what i am on about.
![image](https://github.com/swellstores/community/assets/74880857/cb70946e-df6d-4a1f-a435-bb747e979c78)
![image](https://github.com/swellstores/community/assets/74880857/bdca8c4b-d49b-4c06-9b52-23d62237d569)

I ask this because the url for checkout : https://exoplate-store.swell.store/checkout/test/931c16ffedab9faf675a13fd31e356c4/shipping
seems to work the same with or without the '/test/'.

Which makes me wonder whether or not the connecting a stripe gateway changes the way this reminder email works.

Thanks

**Reply by sags95** [reply]
The gateways are used purely for transactional purposes (creating cards, creating payments, refunds, etc). So the behaviour for abandoned cart or any other emails works the same regardless of the gateway that is connected (even test). When using the Hosted Checkout (above), if they go through the checkout you will see the charge in Stripe along with some other details for the order number.

**Reply by Daniel-Alamezie** [reply]
Ahh so i can connect to stripe but then use swells checkout? When the payment ends up going to my strip account?
Is that what you’re saying?

**Reply by sags95** [reply]
Yes that's correct (or any other gateway).

**Comment by Daniel-Alamezie** [comment]
I added stripe to my swell on the dashboard but when i checkout using the swell custom checkout i still dont see it show up in my stripe dashboard, is there something i am missing?
